ui/src/app/workflows/components/submit-workflow-panel.tsx

```diff
@@ -10,8 +11,10 @@ import {TagsInput} from '../../shared/components/tags-input/tags-input';
 import {services} from '../../shared/services';
 import {Utils} from '../../shared/utils';
 
+type WorkflowTemplateType = 'ClusterWorkflowTemplate' | 'WorkflowTemplate';
+
 interface Props {
-    kind: string;
+    kind: WorkflowTemplateType;
```

Comment:
it would probably make sense to just inline this if you're not using the type anywhere else

Reply:
Sounds good. Done in 59dbea3690b1334d8f9b4cfc1e04ef60cc679141